app/assets/sass/modules/_breadcrumbs.scss

```diff
@@ -0,0 +1,31 @@
+.breadcrumbs {
```

Comment:
I think this file could be simplified by using `.breadcrumbs, #whitehall-wrapper .breadcrumbs` as the selector.

Maybe that's out of scope for this PR though.